status-im/status-react
# [#16077] Add community illustrations

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

fixes #16077

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)

Adds the communtiy illustrations shown when the content  in the tabs is empty according to [designs](https://www.figma.com/file/NAdciBFYSBXmr9wpT47jY1/Illustrations?type=design&node-id=167-65517&t=F7DlRfPtbFAPouaV-4)
![image](https://github.com/status-im/status-mobile/assets/90291778/3ab5e908-22e3-471e-a288-0bc25d6205ec)

![image](https://github.com/status-im/status-mobile/assets/90291778/02e2b894-c395-48b3-ab42-82c4f352eb40)


#### Platforms


- Android
- iOS

### Steps to test


- Open Status
- Navigate to communities tab
- Make sure you have an empty state (no communities joined, pending and opened) to show the new images

status: ready
